crates/bdk/src/wallet/builder.rs

```diff
@@ -0,0 +1,98 @@
+use bdk_chain::PersistBackend;
+use bitcoin::constants::genesis_block;
+use bitcoin::{BlockHash, Network};
+
+use crate::descriptor::{DescriptorError, IntoWalletDescriptor};
+use crate::wallet::{ChangeSet, Wallet};
+
+use super::{InitError, InitOrLoadError};
+
+/// Helper structure to initialize a fresh [`Wallet`] instance.
+pub struct WalletBuilder<E> {
```

Comment:
How do you feel about just calling this `Builder`, i.e. `bdk::wallet::Builder` ?